### PR TITLE
removed dead link: added correct link

### PR DESCRIPTION
**Pull request by RAJU2529**
as per user report #6671 , so i removed dead link and added correct link 
**https://docs.microsoft.com/en-us/powershell/module/skype/?view=skype-ps**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@brandber PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by Kateyanne** [comment]
@JohanFreelancer9  could you check again the new link follows the current protocol?

File: Teams/teams-powershell-managing-teams.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Common parameters:
 
 ## Manage configurations via PowerShell
 
-Find the cmdlets for managing your configuration in the [Skype for Business cmdlet module](/microsoft-365/enterprise/manage-skype-for-business-online-with-microsoft-365-powershell).
+Find the cmdlets for managing your configuration in the [Skype for Business cmdlet module](https://docs.microsoft.com/powershell/module/skype/?view=skype-ps).
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-managing-teams.md:70]:
```suggestion
Find the cmdlets for managing your configuration in the [Skype for Business cmdlet module](/powershell/module/skype/?view=skype-ps).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@Kateyanne - Thank you for picking this one up. I approved the PR on 17 February, before the new protocol came into effect. @RAJU2529 - could you please change the link to a relative link? Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@rajgodwin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tiburd** [comment]
@pbafna03 
Hi, Prashant,
Can you please review this old PR and let me know if you have a replacement link, or if we should remove the entire sentence with the bad link?
Thank you!

**Comment by pbafna03** [comment]
> @pbafna03 Hi, Prashant, Can you please review this old PR and let me know if you have a replacement link, or if we should remove the entire sentence with the bad link? Thank you!

I think a better option for both sections "Manage policies via PowerShell" & "Manage configurations via PowerShell" would be : 

Find the cmdlets for managing your configuration in https://docs.microsoft.com/en-us/powershell/module/teams & https://docs.microsoft.com/en-us/powershell/module/skype.

**Comment by tiburd** [comment]
@brandber 
Hi Brandon,
Please note the validation check suggestions to remove pinning views when the context is not about a particular view. 
Thanks!

**Pull request merged**